### ISSUE TITLE
trigger when it reaches a certain section of the viewport/containment

### ISSUE THREAD

**Issue by gunn-fd** [joshwnj/react-visibility-sensor#166]
Could be a feature, where you can set a height of an area where the trigger for visible happens instead of being the whole viewport or whole containement element.

**Comment by AntonioRedondo** [comment]
Yeah, I also came across with this limitation. It would be very useful if React Visibility Sensor was able to fire some callback given some vertical or horizontal level of the parent child or the whole `<body>` element. Something like:

```jsx
<VisibilitySensor
    onScrollPosition={{
        y: 2000, // in pixels
        x: 500, // in pixels
        fromBody: true,
        callback: (direction) => doSomething(direction)
    }} />
```
* `fromBody` is a boolean that if true the whole `<body>` html element would be considered for the scroll events instead of the parent child.
* `direction` could be a string with `up` and `down` possible values.
